css/style.css

```diff
@@ -63,30 +68,67 @@ figcaption{font-style: italic;}
 }
 /*slider control button*/
 .form-group .views-slideshow-controls-bottom{
-	background-image:url('/sites/default/files/inline-images/slider-controller-bg.png');
+	background-color:#ff7202;
 	min-height: 50px;
 	padding:15px 10px;
 }
 .form-group .views-slideshow-controls-bottom .views_slideshow_controls_text_previous a, 
 .form-group .views-slideshow-controls-bottom .views_slideshow_controls_text_pause a, 
 .form-group .views-slideshow-controls-bottom .views_slideshow_controls_text_next a{
 	padding:10px;
-	border: 2px solid #9eb7f9;
+	border: 2px solid rgba(0,0,0,0.2);
 	color:white;
 	-webkit-border-radius: 5px;
 	-moz-border-radius: 5px;
 	border-radius: 5px;
+	-webkit-transition: all 0.3s ease-in-out 0s;
+	-moz-transition: all 0.3s ease-in-out 0s;
+	transition: all 0.3s ease-in-out 0s;
 }
 .form-group .views-slideshow-controls-bottom .views_slideshow_controls_text_previous{} 
 .form-group .views-slideshow-controls-bottom .views_slideshow_controls_text_pause{}
 .form-group .views-slideshow-controls-bottom .views_slideshow_controls_text_next{}
 
-/*home page dashboard*/
-.home-image-feature{background-color:#FFF;position: relative;}
-.home-image-feature img{width: 100%}
+/*home page - set blocks to match Magento store in colours and padding*/
+.home-bg {margin:15px 0 0 0;}
+.col-md-4 .home-bg, .col-md-8 .home-bg {display: flex;}
+.home-bg .home-image-feature{margin:0 15px 0 0;padding:0;}
+.home-bg .home-image-text{background-color: #13b5ea;margin:0}
```

Comment:
I know we haven't covered this before, but our CSS coding style (https://tartarus.gaiaresources.com.au/wiki_2012/index.php?title=CSS_Style woefully out of date, but it kind of works) says to format css with individual selectors on subsequent lines, and with each property on it's own line. This makes it easy to update individual properties / selectors without creating conflicts in version control. If possible could you reformat this file following that format. Also a space before the opening brace.

Copying the style below in case you can't get to the wiki:

``` css
.class {
    style1: value;
    style2: value;
}

#identifier {
    style1: value;
    style2: value;
}

other css .selector {
    style1: value that is really
        really really really really
        long;
    style2: value;
}

multiple .complex,
#css_selectors .complex {
    style1: value;
}
```